pkg/v1/providers/ytt/vendir/cni/_ytt_lib/addons/packages/antrea/1.2.3/bundle/config/overlay/antrea_overlay.yaml

```diff
@@ -192,14 +206,49 @@ tlsCipherSuites: #@ values.antrea.config.tlsCipherSuites
 #! TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 #!tlsMinVersion:
 
+#! If Antrea is upgraded from version <= v0.13 and legacy CRDs are used, this option should be
+#! enabled, otherwise the CRDs created with the legacy API groups will not take any effect and
+#! work as expected. When the mirroring is enabled, if a legacy CRD is created with legacy API
+#! groups, mirroring-controller will create a new CRD with the Spec and Labels from the legacy
+#! CRD. Afterwards, the modification of Spec and Label in legacy CRD will be synchronized to new
+#! CRD automatically. In addition, the modification of Status in new CRD will also be synchronized
+#! to legacy CRD automatically. If a legacy CRD is deleted, the corresponding new CRD will be deleted.
+#! Note that: to decouple a new CRD from the corresponding legacy CRD, the legacy CRD should be
+#! annotated with "crd.antrea.io/stop-mirror". Afterwards, updates to the legacy CRDs will no
+#! longer be reflected in the new CRD, and all CRUD operations should be done through the new
+#! API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
+#! new CRDs.
+legacyCRDMirroring: true
+
 #! Enable usage reporting (telemetry) to VMware.
-#!enableUsageReporting: false
+#! enableUsageReporting: false
+nodeIPAM:
 
-#! enterpriseAntrea: true
+#! Enable the integrated Node IPAM controller within the Antrea controller.
+#!  enableNodeIPAM: false
 
+#! CIDR Ranges for Pods in cluster. String array containing single CIDR range, or multiple ranges.
+#! The CIDRs could be either IPv4 or IPv6. Value ignored when enableNodeIPAM is false.
+#!  clusterCIDRs: []
+
+#! CIDR Ranges for Services in cluster. It is not necessary to specify it when there is no overlap with clusterCIDRs.
+#! Value ignored when enableNodeIPAM is false.
+#!  serviceCIDR:
+#!  serviceCIDRv6:
+
+#! Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. Value ignored when enableNodeIPAM is false
+#! or when IPv4 Pod CIDR is not configured. Valid range is 16 to 30.
+#!  nodeCIDRMaskSizeIPv4: 24
+
+#! Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. Value ignored when enableNodeIPAM is false
+#! or when IPv6 Pod CIDR is not configured. Valid range is 64 to 126.
+#!  nodeCIDRMaskSizeIPv6: 64
+
+#! enterpriseAntrea: true
```

Comment:
enterpriseAntrea is disabled on CSR bootstrap phase

Reply:
May I know the reason to disable on CSR bootstrap phase?

Reply:
This overlay commenting enterpriseAntrea is made on 0.13.3 and 0.11
https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/providers/ytt/vendir/cni/_ytt_lib/addons/packages/antrea/0.13.3/bundle/config/overlay/antrea_overlay.yaml#L198